packages/block-library/src/cover/editor.scss

```diff
@@ -65,3 +65,42 @@
 	// Important is used to have higher specificity than the inline style set by re-resizable library.
 	height: auto !important;
 }
+
+// Only target direct dropzone.
+.wp-block-cover > .components-drop-zone {
+	&.is-active {
+		transition: 0.3s opacity, 0.3s border;
```

Comment:
Nice. We use 0.2s in most other places to make it snappy — if that feels okay to you, might be a good change. If it feels too snappy, okay to keep 0.3, but worth a shot. 

Might want to include this right below the transition rule:

```
@include reduce-motion("transition");
```

Reply:
I think 0.2s looks better, and it isn't too snappy.

Wasn't aware of the `reduce-motion` mixin, Thanks for mentioning it.

Reply:
Yep, it's easily missed. We have one for the animation property as well, and if you basically output it on the line right after either transition or animation both, you'll be sure that for people to check the system level setting have their needs met. It's a nice little trick, it just sets the duration to about zero.